src/phoenix/server/api/schema.py

```diff
@@ -85,17 +100,18 @@ def node(self, id: GlobalID, info: Info[Context, None]) -> Node:
             embedding_dimension = info.context.model.embedding_dimensions[node_id]
             return to_gql_embedding_dimension(node_id, embedding_dimension)
         elif type_name == "Project":
-            if (traces := info.context.traces) is not None:
-                projects = {
-                    project_id: (project_name, project)
-                    for project_id, project_name, project in traces.get_projects()
-                }
-                if node_id in projects:
-                    name, project = projects[node_id]
-                    return Project(id_attr=node_id, name=name, project=project)
-            raise Exception(f"Unknown project: {id}")
-
-        raise Exception(f"Unknown node type: {type}")
+            async with info.context.db() as session:
+                project = await session.scalar(
+                    select(models.Project).where(models.Project.id == node_id)
+                )
+            if project is None:
+                raise ValueError(f"Unknown project: {id}")
```

Comment:
```suggestion
                if (project := await session.scalar(
                    select(models.Project).where(models.Project.id == node_id)
                )) is None:
                    raise ValueError(f"Unknown project: {id}")
```
Nit

Reply:
it's more lines, not fewer

<img width="318" alt="Screenshot 2024-04-12 at 10 40 20 AM" src="https://github.com/Arize-ai/phoenix/assets/80478925/22b85fc7-72f4-4858-9b10-9cbf588b16f1">